docs/start/qiskit-versioning.mdx

```diff
@@ -0,0 +1,135 @@
+version. Shortly before a new major version, a final minor version will
+be published. This final minor version release `M.N+1.0` is equivalent to
+`M.N.0` but with warnings and deprecations for any API changes that are
+made on the new major version series.
+
+For example, immediately proceeding the 1.0.0 release, a 0.46.0 release will be
+published. The 0.46.0 release will be equivalent to the 0.45.0 release but with
+additional deprecation warnings that document the API changes that were made as
+part of the 1.0.0 release. This pattern will be used for any future major
+version releases.
+
+As a user of Qiskit, it's recommended that you first upgrade to this final minor
+version first, so you can see any deprecation warnings and adjust your Qiskit
+usage ahead of time before trying a potentially breaking release. The previous
+major version will be supported for at least six months to give sufficient time
+to upgrade. A typical pattern to manage this is to pin the max version to
+avoid using the next major release series until you're sure of compatibility.
+For example, specifying in a requirements file `qiskit<2` when the current
+major Qiskit version is 1 will ensure that you're using a version of Qiskit
+that won't have breaking API changes.
+
+Pre-emptively capping the version less than the next major version is necessary
+to ensure you get a chance to see deprecation warnings before a
+major version release. The normal release schedule means the last minor
+version release which includes any final deprecation warnings will be released
+shortly before the next major version and `pip` will default to using
+the newest version available unless the version cap is set.
+
+## Pre-releases
+
+For each minor and major version release Qiskit will publish pre-releases that
+are compatible with [PEP440](https://peps.python.org/pep-0440/). Typically
+these are release candidates of the form `X.Y.0rc1`. The `rc` releases
+will have a finalized API surface and are used to test a prospective release.
+
+Note that when one of the PEP440 pre-release suffixes (such as `a`, `b`, or `pre`) are
+published, it does not have the same guarantees as an `rc` release, and is
+only a preview release. The API might change between these pre-releases
+and the final release with that version number. For example, `1.0.0pre1` might have
+a different final API than `1.0.0`.
+
+## Post-releases
+
+If there are issues with the packaging of a given release, a post-release may be
+issued to correct this. These will follow the form `X.Y.Z.1` where the fourth
+integer is used to indicate it is the first post-release of the `X.Y.Z` release.
+For example, the qiskit-terra (the legacy package name for Qiskit) 0.25.2
+release had some issue with the sdist package publishing, and a post-release
+0.25.2.1 was published that corrected this issue. The code was identical, and
+0.25.2.1 only fixed the packaging issue for the release.
+
+
+## Next steps
```

Comment:
I think the only topics that apply as next steps would be Install and set up Qiskit, and Configure Qiskit locally.  And maybe Release notes (https://docs.quantum.ibm.com/api/qiskit/release-notes)